clang/test/Preprocessor/sycl-macro.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: %clang_cc1 %s -E -dM | FileCheck %s
 // RUN: %clang_cc1 %s -fsycl-is-device -E -dM | FileCheck --check-prefix=CHECK-SYCL-ID %s
-// RUN: %clang_cc1 %s -fsycl-id-queries-fit-in-int -fsycl-is-host -sycl-std=2017 -E -dM | FileCheck --check-prefix=CHECK-SYCL-STD %s
-// RUN: %clang_cc1 %s -fsycl-id-queries-fit-in-int -fsycl-is-device -sycl-std=2017 -E -dM | FileCheck --check-prefix=CHECK-SYCL-STD %s
```

Comment:
We already have `RUN` lines for SYCL-2020, I don't think that we need more of them. Lets just drop these two and update `CHECK-SYCL-STD-2020` check lines to include `__SYCL_ID_QUERIES_FIT_IN_INT__`